packages/graph/clients/typescript/api.ts

```diff
@@ -4396,7 +4396,7 @@ export const GraphApiFactory = function (
      * @param {*} [options] Override http request option.
      * @throws {RequiredError}
      */
-    getEntityLinks(entityId: string, options?: any): AxiosPromise<object> {
+    getEntityLinks(entityId: string, options?: any): AxiosPromise<Array<Link>> {
```

Comment:
I believe this will break some parts of the workspace API. I'm okay with it being broken, as I'll be working on changing the links right away. Are we okay with the links not functioning? Alternatively we merge this PR into the branch https://github.com/hashintel/hash/tree/asa/only-multi-links instead to make sure we don't break any type editor work :)

Reply:
I applied a fix in ca5f75aea59bc40141a06ae331ba29db7ae7428d